config/index.js

```diff
@@ -8,7 +8,16 @@ const activities = {
   exit: `Screw u guys I'm going home`,
 };
 
+const botToken = process.env.BOT_TOKEN || '';
```

Comment:
What about having a `config` object with all of these properties, instead?

Advantages are not having loose constants scattered across the codebase and it's easier if we use `config.` and the IDE should auto-complete, so that we don't have to remember everything.

Reply:
Check my last commit.